doc/source/data_structures.rst

```diff
@@ -13,14 +13,14 @@
 Data Structures
 =========================================
 
-GeoPandas implements two main data structures, a ``GeoSeries`` and a
-``GeoDataFrame``.  These are subclasses of pandas ``Series`` and
+GeoPandas implements two main data structures, a :class:`~geopandas.GeoSeries` and a
```

Comment:
All the `~geopandas.`'s should not be needed (as geopandas is the "currentmodule)

Reply:
I got rid of most of them, but left a couple so only the methods show up to be consistent with how the docs were previously.